tests/test_transform.py

```diff
@@ -93,12 +87,9 @@ def test_search_space_transform_encoding() -> None:
         (1, IntDistribution(1, 10, log=True)),
         (2, IntDistribution(0, 10, step=2)),
         (10, IntUniformDistribution(0, 10, step=2)),
-        (0.0, UniformDistribution(0, 3)),
-        (3.0, UniformDistribution(0, 3)),
-        (1.0, LogUniformDistribution(1, 10)),
-        (10.0, LogUniformDistribution(1, 10)),
-        (0.2, DiscreteUniformDistribution(0, 1, q=0.2)),
-        (1.0, DiscreteUniformDistribution(0, 1, q=0.2)),
+        (3.0, FloatDistribution(0, 3)),
+        (10.0, FloatDistribution(1, 10, log=True)),
+        (1.0, FloatDistribution(0, 1, step=0.2)),
         (0.0, FloatDistribution(0, 3)),
         (1.0, FloatDistribution(1, 10, log=True)),
         (0.2, FloatDistribution(0, 1, step=0.2)),
```

Comment:
```suggestion
        (0.0, FloatDistribution(0, 3)),
        (3.0, FloatDistribution(0, 3)),
        (1.0, FloatDistribution(1, 10, log=True)),
        (10.0, FloatDistribution(1, 10, log=True)),
        (0.2, FloatDistribution(0, 1, step=0.2)),
        (1.0, FloatDistribution(0, 1, step=0.2)),
```

How about align group by distribution?

Reply:
Done in 8591eda0f59c0b256023653c8b9d1f1ea2b4a6df.